### ISSUE TITLE
Additional fields in raw status for monitoring purpose

### ISSUE THREAD

**Issue by dcdamien** [zrepl/zrepl#499]
Currently `zrepl status --mode raw` is a bit complex making it hard to parse using monitoring systems other than Prometheus.
It will be easier to monitor with additional fields like LastAttemptState and LastSuccessfulAttempt (date) for repllication and similarly for the others tasks like pruning.

**Comment by problame** [comment]
What exactly do you find difficult to parse? For any `LastAttempt*` value, you'll have to use the last entry in the attempts array.
